ush/python/pyobsforge/obsdb/obsdb.py

```diff
@@ -0,0 +1,100 @@
+import sqlite3
+from datetime import datetime, timedelta
+import sys
+import os
+# TODO(G): Hack to import wxflow module, do this properly.
+sys.path.append(os.path.join(os.path.dirname(__file__), '../../../../sorc/wxflow/src/'))
```

Comment:
why not just set the PYTHONPATH in the test environment?

Reply:
remove it from here and define PYTHONPATH in pytest.yaml before running the tests